### PR TITLE
feat(Functions): Add ref content for billing with link to FAQ MTA-3306

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: serverless/functions/reference-content/functions-billing.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  posted: 2023-06-15
+categories:
+  - serverless
+---
+
+Serverless Functions are billed on a pay-as-you-go basis depending on: 
+
+* Monthly Request number
+* Resource consumption
+* Resources provision
```
Comment (lucasscw) [inline serverless/functions/reference-content/functions-billing.mdx:20]:
```suggestion
* **Monthly Request number:** each time your function is invoked we increase a counter.
* **Resource consumption:** this component is obtained by multiplying the memory tiers chosen (128MB, 256MB, 512MB, 1024MB, 2048MB) by the duration of each function invocation.
* **Resources provision:** in order to mitigate cold start, users can choose to keep some function Instances warm (by filing the min scale value). We then charge for the provisioned resources similarly to the Resources consumption component.

The scheme below illustrates our billing model:

<Lightbox src={["scaleway-Billing-FaaS-1-DEF.webp", "scaleway-Billing-FaaS-2-DEF.webp"]} size={"medium"}  alt={["Functions Billing", "Functions Billing"]} />
```

Comment (lucasscw) [reply serverless/functions/reference-content/functions-billing.mdx:20]: What do you think about taking more content from the FAQ, IMHO customers expect to find at least some detailed information right away, it is painful to have to click on another link a leave the current section of the documentation to get some info

**Review by lucasscw — CHANGES_REQUESTED** [review]

File: serverless/functions/reference-content/functions-billing.mdx (hunk 1 above)
Comment (SamyOubouaziz) [reply serverless/functions/reference-content/functions-billing.mdx:20]: Ok.

Comment (SamyOubouaziz) [reply serverless/functions/reference-content/functions-billing.mdx:20]: As discussed, Billing stays in FAQ, with a link from Ref content to it.

**Review by bene2k1 — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**